xbmc/video/ContextMenus.cpp

```diff
@@ -33,9 +33,13 @@ CVideoInfo::CVideoInfo(MediaType mediaType)
 
 bool CVideoInfo::IsVisible(const CFileItem& item) const
 {
-  if (item.IsPVR())
-    return false; // pvr has its own implementation for this
-  return item.HasVideoInfoTag() && item.GetVideoInfoTag()->m_type == m_mediaType;
+  if (!item.HasVideoInfoTag())
+    return false;
+
+  if (item.IsPVRRecording())
```

Comment:
Okay, please also formally approve the Krypton backport of the PR, otherwise I cannot merge it. Thx.